<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glaxnimate::gui::ResizeDialog</class>
 <widget class="QDialog" name="ResizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resize Document</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="check_aspect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Lock Apect Ratio</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset theme="lock">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="check_scale_layers">
     <property name="text">
      <string>Scale Layers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="check_layer_ratio">
     <property name="text">
      <string>Preserve Layer Aspect Ratio</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spin_width">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spin_height">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResizeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResizeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_scale_layers</sender>
   <signal>toggled(bool)</signal>
   <receiver>check_layer_ratio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ResizeDialog</receiver>
   <slot>height_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_width</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ResizeDialog</receiver>
   <slot>width_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_aspect</sender>
   <signal>clicked(bool)</signal>
   <receiver>ResizeDialog</receiver>
   <slot>lock_changed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>width_changed(int)</slot>
  <slot>height_changed(int)</slot>
  <slot>lock_changed(bool)</slot>
 </slots>
</ui>
